src/ui/public/vis/__tests__/_agg_config.js

```diff
@@ -481,12 +481,12 @@ describe('AggConfig', function () {
           {
             type: 'avg',
             schema: 'metric',
-            params: { field: 'ssl' }
+            params: { field: 'bytes' }
           }
         ]
       });
 
-      let field = indexPattern.fields.byName.ssl;
+      let field = indexPattern.fields.byName.bytes;
```

Comment:
That was incorrect, I misread the field list and miscategorized the issue. Removed the changes.